core/src/i18n/pl.yaml

```
@@ -31,19 +31,24 @@ common:
   buttons:
     cancel: Anuluj
     delete: Usuń
+    edit-yaml: Edytuj YAML
+  created: Stworzono
   delete-dialog:
     delete-confirm: Nie pytaj ponownie
     message: Czy na pewno chcesz usunąć {{type}} {{name}}?
     title: Usuń {{name}}
+  labels: Etykiety
+  name: Nazwa
+  status: Status
   tooltips:
-    download: Pobierz
-    search: Szukaj
-    save: Zapisz
     copy-to-clipboard: Skopiuj do schowka
+    download: Pobierz
     duplicate-key: Zduplikowany klucz
-    k8s-name-input: Nazwa musi zawierać małe znaki alfanumeryczne, może zawierać „-” (np. „my-name1”).
+    k8s-name-input: Nazwa musi składać się z małych znaków alfanumerycznych, może zawierać „-” (np. „my-name1”).
```

#nitpick
I'd add "również", because this is extra info, as "-" is not an alphanumeric character.
```suggestion
    k8s-name-input: Nazwa musi składać się z małych znaków alfanumerycznych, może zawierać również „-” (np. „my-name1”).
```